portal/src/main/webapp/js/src/study-view/component/PieChart.js

```diff
@@ -206,7 +209,9 @@ var PieChart = function(){
         });
 
         if(category[1] === 'h1' && currentView === 'pie') {
-            initMainDivQtip();
+            $('#' + DIV.mainDiv).one('mouseover', function () {
```

Comment:
I would put this check in the initMainDivQtip() itself. See https://github.com/thehyve/cbioportal/commit/c4ee712310cb9c5824fb0d74ccb79812f0b75cf6